README.md

```diff
@@ -60,6 +60,34 @@ An initial run_list for the node can also be passed to the subcommand. Example u
 
     knife bootstrap windows ssh ec2-50-xx-xx-124.compute-1.amazonaws.com -r 'role[webserver],role[production]' -x Administrator -i ~/.ssh/id_rsa
 
+### knife windows cert generate
+
+Generate a certificate(x509) containing a public / private key pair for WinRM 'SSL' communication.
```

Comment:
It should be "Generates a certificate(x509) containing public/private key pair for WinRM 'SSL' communication."